packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -125,7 +134,32 @@ export function FormFieldWrapper({
           )}
 
           {prefix?.label && <AffixLabel {...prefix} labelRef={prefixRef} />}
-          <div className={styles.childrenWrapper}>{children}</div>
+
+          <div className={styles.childrenWrapper} tabIndex={-1}>
+            {children}
+            <AnimatePresence>
+              {isToolbarVisible && (
+                <motion.div
```

Comment:
Oh i just realized something. This toolbar shouldn't be here actually. It should be on below everything else as it will mess with the positioning of prefix and suffix.

Moving it down like components-native should fix the tabbing order tho

Web

![image](https://github.com/GetJobber/atlantis/assets/15986172/6cef2251-0ec3-4f94-83ef-94e5a0cead57)


Components native

![image](https://github.com/GetJobber/atlantis/assets/15986172/5de3557a-b2dd-4228-8d71-8fd4a6c56ccf)

Reply:
when you say "everything else" what do you mean? there are a few different levels 

- outside the entire `containerClasses` div
- outside the `wrapperClasses` div
- outside the `inputWrapper` div?

Reply:
thinking somewhere inside the wrapperClasses

Reply:
this was a wild ride but I think it's done with the prefixes not totally busted (for single line, multiline is already borked)